docs/tasks/issuers/setup-acme/dns01/route53.rst

```diff
@@ -22,6 +22,8 @@ Amazon Route53
              secretAccessKeySecretRef:
                name: prod-route53-credentials-secret
                key: secret-access-key
+             # you may specify a role to assume
+             role: arn:aws:iam::XXXXXXXXXXXX:role/cert-manager
```

Comment:
I think a full blown example would be great here (Including IAM Role configuration, AssumePolicy and so on)

Also pointing out the two options using Instance Profile + Assume Role (`useAmbientCerds=false`) or AccessKey + SecretKey + AssumeRole. (`useAmbientCerds=true`)

Reply:
Yep, are these to auth methods mutually exclusive? If you could update comments on API types to explain this too, we *may* want to consider restructuring how auth is defined if so to make it fit better.. but let's first document what's what so we are all on the same page 😄